model/riscv_vmem_tlb.sail

```diff
@@ -36,100 +73,98 @@ type tlb_index_range = range(0, num_tlb_entries - 1)
 register tlb : vector(num_tlb_entries, option(TLB_Entry)) = vector_init(None())
 
 // Indexed by the lowest bits of the VPN.
-function tlb_hash(vaddr : bits(64)) -> tlb_index_range =
-  unsigned(vaddr[17 .. 12])
+function tlb_hash forall 'v, is_sv_mode('v) . (
+  sv_mode : int('v),
+  vpn     : vpn_bits('v),
+) -> tlb_index_range =
+  unsigned(vpn[5 .. 0])
 
-// PUBLIC: invoked in init_vmem() [riscv_vmem.sail]
-function init_TLB() -> unit = {
```

Comment:
Will we not need 'init_TLB()' if we allow a dynamic reset on the Sail model?

Reply:
Good question. We don't for RVFI since the TLB will automatically be reset by Sail. For "real" resets I guess *technically* you don't need to reset the TLB for a functioning chip. Feels like that would be an unnecessarily risky design though. I'll add it back in.

Reply:
Added back.